frontend/app/src/views/source-data/ProcessControlView.vue

```diff
@@ -167,9 +167,9 @@ export default {
       // Get the header for this table
       await this.fetchFormFieldHeaders();
       // Get the headers from user preferences
-      await this.fetchUserPreferenceByColumnType(ColumnTypes.PROCESSVIEW);
-      if (this.userPreferenceData.length) {
-        this.onlyShowColumns = this.userPreferenceData;
+      await this.fetchUserPreferenceByViewName(ColumnTypes.PROCESSVIEW);
+      if (this.displayColumnsData.length) {
+        this.onlyShowColumns = this.displayColumnsData;
```

Comment:
did you miss here the name?

Reply:
Yes I think , But now it's renamed to fetchUserPreference()